### DISCUSSION TITLE
다크모드에서 글자가 흰색으로 출력되지 않습니다

### DISCUSSION THREAD

**Discussion by theif45** [pronist/hELLO#208] [Q&A] [answered]
아래 이미지와 같이 다크모드에서 글자색도 회색이나 검은색으로 출력되고 배경도 흰색으로 나올때가 있습니다 도저히 어떻게 해결해야 될 지 몰라서 문의드립니다.
![Screenshot_1](https://user-images.githubusercontent.com/121926183/228098198-2b3bfab0-240c-40a1-9e7c-473f02210dbc.png)

**Comment by pronist** [comment, marked as answer]
@theif45 안녕하세요.

글을 작성하실 때 다른 곳에서 복사&붙여넣기 하신 경우 티스토리 에디터에서 직접 작성한 것과는 다르게 해석됩니다. 다크모드에서 색상이 바르게 출력되지 않는 경우는 대부분 복붙을 한 경우이므로 문제가 생긴 글에서 HTML 모드로 진입하여 색상이 지정된 텍스트를 수정해야합니다. 가장 나은 방법은 복붙이 아니라 티스토리 에디터로 작성하는 것입니다. 이 스킨은 티스토리 에디터로 직접 작성한 글에 대해 대응하고 있습니다.

**Reply by theif45** [reply]
감사합니다 직접 작성하니 정상적으로 나오네요
